src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -2,7 +1,20 @@
+import React from 'react';
+
+type Props = {
+  handlerSelect: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  handlerInput: (event: React.ChangeEvent<HTMLInputElement> | string) => void;
+  query: string;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  handlerSelect: handlerFilter,
+  handlerInput: handlerInput,
```

Comment:
Consider renaming `handlerFilter` and `handlerInput` to `handleSelectChange` and `handleInputChange` respectively, to follow common naming conventions for event handlers and improve code readability.